### ISSUE TITLE
 Error if the execution path contains spaces(windows)

### ISSUE THREAD

**Issue by Labratorite** [vorburger/MariaDB4j#346]
Hi ,
I'm trying to use MariaDB4j with spring boot on windows server.
```
mariaDB4j:
  dataDir: ./localdb/data
  baseDir: ./localdb/base
```
The directory I'll provide are variable and the directory names can't specify.
However an error occurred run mysqld.exe if the working path contained spaces.

I know my  environment is not smart, but I would appreciate it if you fix it.

**Comment by Ejiik** [comment]
Hi, I'm having the same problem. 
Getting a "ch.vorburger.exec.ManagedProcessException: An error occurred while installing the database" 
with a "FATAL ERROR: Cannot set current directory to 'C:\Users\Erik'". (dir folder name is "Erik Johansson", so the space messes it up, I'm guessing) 
Would also appreciate a fix, thanks. 😃

**Comment by Kilobyte1000** [comment]
Caused by processBuilder automatically adding double quotes. Detailed in [this SO answer](https://stackoverflow.com/a/20009602/12656386)

**Comment by vorburger** [comment]
This is fixed on the code in the development branch, but that hasn't been released; watch #334.

**Comment by vorburger** [comment]
v2.5.0 is now released, so closing this issue.
